MaterialDesignExtensions/Localization/Strings.uz-Latin-UZ.resx

```diff
@@ -162,6 +162,9 @@
   <data name="LocalDrive" xml:space="preserve">
```

Comment:
Why renaming this file? I checked the string and uz-Latn-UZ is the correct language tag (see [RFC 4646](https://tools.ietf.org/html/rfc4646) and [RFC 4647](https://tools.ietf.org/html/rfc4647)). Merging this change may break the support for the language.